code/modules/admin/admin_verbs.dm

```diff
@@ -2215,3 +2215,17 @@ var/list/fun_images = list()
 				if((i++ % 5) == 0)
 					sleep(1 SECOND)
 				apc.setStatus("lightsout", dur SECONDS)
+
+/client/proc/flock_cheat()
+	SET_ADMIN_CAT(ADMIN_CAT_DEBUG)
+	set name = "Flock infinite compute"
+	set desc = "Toggle infinite compute on or off on a particular flock"
+
+	var/flockname = tgui_input_list(src, "Pick a flock", "Choose flock", flocks)
```

Comment:
Could this also have another menu for which cheat to enable/disable - if we're doing multiple?

Mostly I'm just thinking we should have a cheat for "unlock all the buildings" as well as a cheat for "infinite compute".